identity/webapp/next.config.js

```diff
@@ -0,0 +1,4 @@
+const webpack = require('webpack');
+const config = require('@weco/common/next/next.config');
```

Comment:
I think it may well be easier to use a separate `next.config` for this app for now - the common one ties us to some things which we don't need and maybe would do differently now, so even if it's just a lot of copy-and-paste I think keeping it as separate as possible is best

Reply:
Additionally I think the rewrites/redirects we have will interact badly with this